### ISSUE TITLE
ETA for a full linux release

### ISSUE THREAD

**Issue by lopezator** [TablePlus/TablePlus-Linux#77]
Is there any ETA for a full linux release?

I am a paid user who switched from MacOs to linux, and as-is this alpha is unusable for me, I'd rather pay for a full linux release (in sync with MacOs and Windows releases).

**Comment by ZayRTun** [comment]
Yes is there any news on a release? I am willing to be a paid user and I am looking forward to it.

**Comment by huyphams** [comment]
We're trying to release it this year (2 months left)

**Comment by ZayRTun** [comment]
Glad to know that it's coming soon, really looking forward to it.

**Comment by appcoders** [comment]
Awesome looking forward to it as I bought it on Mac and iOS.

**Comment by scheMeZa** [comment]
Also bought licenses and wanting to switch to ubuntu. Looking forward to the release! 👍

**Comment by lspw** [comment]
> We're trying to release it this year (2 months left)

@huyphams  So, is it happening or maybe in the early months of 2021?

**Comment by MohammadAbusaleh** [comment]
Hello @huyphams, any update on this ?

**Comment by scheMeZa** [comment]
Any news?

**Comment by gdonoso94** [comment]
Hi, any update on this?

**Comment by lopezator** [comment]
We just renewed our team licenses to support the project, but as part of
our team switched to Linux, we might consider not doing it next year if the
situation persists for so long.

Any update would be welcome!

Thanks.

**Comment by huyphams** [comment]
Hi guys, we are sorry about the delay. Currently we are working on the export function, hopefully we can release this feature within two week.

@lopezator thank you very much. I really appreciate your support!

**Comment by lopezator** [comment]
> Hi guys, we are sorry about the delay. Currently we are working on the export function, hopefully we can release this feature within two week.
> 
> @lopezator thank you very much. I really appreciate your support!

You're welcome. I appreciate the update, and thank you for your hard work.

**Comment by scheMeZa** [comment]
> Hi guys, we are sorry about the delay. Currently we are working on the export function, hopefully we can release this feature within two week.
> 
> @lopezator thank you very much. I really appreciate your support!

Thank you so much for the update! I'm in the same boat with my team.

**Comment by diogogmt** [comment]
Is there an issue or website with a matrix showing what features are missing from the Linux version?
Specifically, I'm looking for the table overview and snippet management features, any ETA when they will be available?

Thanks!

**Comment by gdonoso94** [comment]
Hi,

Is there any update on this?

**Comment by mroeling** [comment]
Speed so far is great! I can see a promising future.
But it still not feels like feature complete. Any e.t.a. on this?
